### PR TITLE
Improve .mailmap

### PR DESCRIPTION
**Pull request by ADmad**
Simplified existing mappings and added few more.

`git shortlog -sne` now returns **591** lines.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -22,111 +22,75 @@ Mark Scherer <euromark@web.de> mscherer <mark.scherer@spryker.com>
...
 Walter Nasich <wnasich@gmail.com>
-Walter Nasich <wnasich@gmail.com> wnasich <wnasich@gmail.com>
 mstra001 <ms@creemedia.com>
-mstra001 <ms@creemedia.com> mathieu strauch <ms@creemedia.com>
 Aymeric Derbois <aymeric@derbois.com>
-Aymeric Derbois <aymeric@derbois.com> Derbois Aymeric <aymeric@derbois.com>
 Daniel <wrightdaniel86@gmail.com>
-Daniel <wrightdaniel86@gmail.com> dangerdan <wrightdaniel86@gmail.com>
 James Michael DuPont <JamesMikeDuPont@gmail.com>
-James Michael DuPont <JamesMikeDuPont@gmail.com> James Michael DuPont <jamesmikedupont@gmail.com>
```
Comment (dereuromark) [inline .mailmap]: I dont think this is valid.
It duplicates it again for me.

**Comment by dereuromark** [comment]
Interesting, so the user names are not necessary, but the casing of email matters.
Did you adjust the shell script?

I added a commit on top cutting it down to **583** (if you undo your invalid removal above).

File: .mailmap (hunk 1 above)
Comment (ADmad) [inline .mailmap]: Yeah this was a mistake. Do you mind adding it back since you are already doing other changes.

**Comment by dereuromark** [comment]
582 now.

**Comment by ADmad** [comment]
> Interesting, so the user names are not necessary, but the casing of email matters. Did you adjust the shell script?

Yeah mainly the email matters. If someone has 2 emails and same name you don't even need 2 lines in the mailmap, just a single line `name <email1> <email2>` suffices. I didn't make any changes to your script. Did it manually with a keen eye :)

**Pull request merged**